pkg/apis/clusterregistry/v1alpha1/generated.proto

```diff
@@ -74,6 +74,7 @@ message AuthProviderType {
 }
 
 // Cluster contains information about a cluster in a cluster registry.
+// +k8s:openapi-gen=x-kubernetes-print-columns:custom-columns=NAME:.metadata.name,URL:.spec.url,STATUS:.status.conditions[*].reason
```

Comment:
I think a start here would be:

- name as `NAME`
- relevant information about the endpoints

When it comes to endpoints, I'm not sure what users would want to see:

- The list of the actual endpoints themselves is probably part of the story
- I don't think just a list of CIDRs is what people would want
- A CIDR->address mapping is probably most useful, but I don't think the openapi column feature can give us `spec.field.fieldA -> spec.field.fieldB` type formatted outputs

Reply:
cc @font @perotinus @luksa

Reply:
Made some changes to make the openapi spec more in line with this--but the issue is not resolved. Also, getting the AGE doesn't seem to be possible right now (or I haven't been able to figure it out) and depends on kubernetes/kubernetes#56301 being merged. I have left `Creation Time` as a stand in for age.
Example output:
```kubectl get clusters
NAME            CIDR        SERVER                       CREATION TIME
test-cluster1   0.0.0.0/0   https://192.168.39.97:8443   2018-03-19T23:12:19Z
test-cluster2   0.0.0.0/0   https://192.168.39.97:8443   2018-03-19T23:12:20Z
test-cluster3   0.0.0.0/0   https://192.168.39.97:8443   2018-03-19T23:12:20Z
```